editor/test/integration/app_view_test.clj

```diff
@@ -166,3 +169,55 @@
         (is (seq (:artifacts build-results)))
         (is (not (g/error? (:error build-results))))
         (workspace/artifact-map! workspace (:artifact-map build-results))))))
+
+(deftest build-targets-remain-in-cache-after-build
+  (let [project-path "test/resources/build_project/SideScroller"
+        cache-size 50
+        retained-labels #{:build-targets}]
+    (letfn [(cached-endpoints []
+              (into (sorted-set)
```

Comment:
I'm not sure we need to make endpoints comparable and use a sorted set here. From what I see, it's used for `subset?` and `contains?` check that don't need set ordering. Or are there other uses that I missed?